src/inspector/view/qml/MuseScore/Inspector/notation/lines/internal/LineStyleSection.qml

```diff
@@ -70,7 +72,7 @@ Column {
 
     FlatRadioButtonGroupPropertyView {
         id: styleSection
-        titleText: qsTrc("inspector", "Style")
+        titleText: qsTrc("inspector", root.propertyPanelTitle)
```

Comment:
Unfortunately, this way the translation system can't detect the string (it's not very smart). So, you should do:
```qml
property string propertyPanelTitle: qsTrc("inspector", "Line style")

…

titleText: root.propertyPanelTitle
```

Also, I propose to rename the property to something slightly more specific. What about `lineStyleTitleText`?

Reply:
Thanks for the catch! `lineStyleTitleText` sounds good, will make the changes